### ISSUE TITLE
Unable to activate Focus mode manually if its relevant check boxes are unchecked in the Browse mode section of NVDA settings

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#11030]
### Steps to reproduce:
1. Open NVDA settings, go to the Browse mode section, and make sure both Automatic focus mode for focus changes and Automatic focus mode for caret movement check boxes are unchecked. This means that you should activate manually by, say, pressing Enter on an Edit box.
2. Go to web.telegram.org and in case you happen to have a Telegram account, log in with your number.
3. When logged in, select one of your messages or contacts, and try activating the Focus mode on the Message edit box, by pressing Enter or Space on it, to start typing something to that person, group, etc. The Edit box is toward the bottom of the page, above a button which has no name but which acts as the Send button.
### Actual behavior:
Pressing Enter or Space on the Edit box doesn't put NVDA into Focus mode.
### Expected behavior:
Pressing Enter or Space on the Edit box should activate Focus mode to help users type into it.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
alpha-20005,14c2e2ec
#### Windows version:
Windows 10 Pro 64-Bit, Version 2004
#### Name and version of other software in use when reproducing the issue:
Google Chrome Stable 81.0.4
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Tested with 2019.3 and 2019.3.1 with the same results. It is worth mentioning that the issue cannot be duplicated on many pages, but the Telegram Web example is always duplicable.
#### If addons are disabled, is your problem still occuring?
Yes.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No, as it isn't presumably relevant.

**Comment by amirsol81** [comment]
Wish I could, but I don't have access to Firefox for testing.

**Comment by zahra21** [comment]
hi.
i dont know whether its specific for a particular browser or particular website.
but i wish to remind the hotkey for activating brows mode or focus mode in nvda, is nvda plus space, not enter or space like in jaws!
i hope that it helps you.

**Comment by amirsol81** [comment]
Hi,

When you want to start typing into an edit box in NVDA, you have to press Enter or Space on it. What you mentioned re Insert+Space has nothing to do with typing -- it's more of a navigation concept. It's similar to Insert+Z in JAWS. My reported issue affects entering the text box with Enter or Space when focused on the second edit box of the Telegram Web interface.

**Comment by Adriani90** [comment]
@feerrenrut in Firefox 75 everything works correctly.

I cannot reproduce this issue in Google Chrome 81.0.4044.122
But I can definitely reproduce it in Edge Chromium 81. Pressing enter on an edit field does not trigger the focus mode there.

**Comment by amirsol81** [comment]
This is still duplicable with both Google Chrome 84 and MS Edge Chromium 84.

**Comment by amirsol81** [comment]
This has been resolved, so I'm closing the issue.
